model/flow/chunk.go

```diff
@@ -1,5 +1,9 @@
 package flow
 
+import (
+	"github.com/ipfs/go-cid"
```

Comment:
Just to confirm this is the package we were planning to use, I'm a bit worried that major entities on flow be dependent on the external libs, given that a simple change of order of fields or other facts would impact us heavily

Reply:
This is the package used by bitswap for the data IDs. IPFS also uses it as the address for their data blocks. The core data structure is just a string and the serialization is versioned: https://github.com/ipfs/go-cid/blob/d900d766aff6fc5b91ba8d74efc99ae672701e84/cid.go#L170-L173

With that said, this is a fair point. We could convert these into `flow.Identifier` before adding it to the model, but I don't think would address the concern here. This PR does also add some tests which should catch basic changes:
https://github.com/onflow/flow-go/pull/4586/files#diff-1b92b673833800e45ffd8a743b0958aae4c05a9bfc0ad1780cb70a63c97e4c6dR124-R173